src/components/Speciality/Intro.js

```diff
@@ -17,22 +19,39 @@ const StyledBlueBackground = styled(BlueBackground)`
   margin-top: -${remcalc(36)};
 `
 
+const StyledIntroRectangleInner = styled.div`
+  padding-top: ${({ theme }) => theme.space[3]};
+  padding-bottom: ${({ theme }) => theme.space[3]};
+
+  ${breakpoint('smallPhone', 'smallTablet')`
```

Comment:
you shouldn't need to  specify smallTablet here as the media queries go up

Reply:
*With 'smallTablet'*
![image](https://user-images.githubusercontent.com/31079643/60582806-686fa500-9d81-11e9-9f41-70bad6335a3e.png)

*Without*
![image](https://user-images.githubusercontent.com/31079643/60582865-85a47380-9d81-11e9-9e59-b2e96b67680d.png)

Reply:
Mmh. That's strange. I guess some other style applies on smallTablet then 🤔